src/App.tsx

```diff
@@ -1,26 +1,45 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { selectCurrentTodo, setTodos } from './features';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const [isloading, setIsLoading] = useState(false);
+  const dispatch = useAppDispatch();
+  const selectedTodo = useAppSelector(selectCurrentTodo);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  useEffect(() => {
+    setIsLoading(true);
+
+    getTodos()
+      .then(response => {
+        dispatch(setTodos(response));
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, [dispatch]);
```

Comment:
The `useEffect` dependency array should include all variables that are used inside the effect and are not constants or functions defined outside the component. In this case, `getTodos` is used inside the effect, and if it is not a constant or a function defined outside the component, it should be included in the dependency array.